integreat_cms/cms/templates/events/event_form_sidebar/actions_box.html

```diff
@@ -0,0 +1,49 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load content_filters %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    wrench
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Actions" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div>
+        {% if event_form.instance.archived %}
+            <label>{% translate "Restore event" %}</label>
```

Comment:
```suggestion
            <label class="mt-0">{% translate "Restore event" %}</label>
```
This would e.g. be a label which is the first one in the box and the top margin is larger than the padding to all other sides of the box.